packages/api/src/routes/medical/internal-cq.ts

```diff
@@ -92,16 +93,18 @@ router.post(
   upload.single("file"),
   asyncHandler(async (req: Request, res: Response) => {
     const file = req.file;
-    if (!file) {
-      throw new BadRequestError("File must be provided");
-    }
-    const bundle = JSON.parse(file.buffer.toString());
+    if (!file) throw new BadRequestError("File must be provided");
+    const bundle = JSON.parse(file.buffer.toString()) as Bundle;
```

Comment:
I know we were not doing this before, but don't we want to validate the bundle?

We have a shell for this on `parseFhirBundle()`, and could look into improving it by calling `validateFhirEntries()`.

Defs not a blocker, but could at least map the improvement if it becomes a scope creep.

Reply:
Yea... is there maybe something from medplum so I don't need to do it myself?